CHANGELOG.unreleased.md

```diff
@@ -30,3 +31,7 @@
 > - major: if the change breaks compatibility
 >
 > In case of conflict, the highest (lowest in previous list) `$version` wins.
+
+- @xen-orchestra/backups minor
```

Comment:
```suggestion
- @xen-orchestra/backups minor
- @xen-orchestra/proxy minor
```

Reply:
IIRC, backups on remotes bound to a proxy are restore by this proxy.

Have you checked that you PR handle this?